x/concentrated-liquidity/lp.go

```diff
@@ -298,6 +328,35 @@ func (k Keeper) initializeInitialPositionForPool(ctx sdk.Context, pool types.Con
 	return nil
 }
 
+// uninitializePool uninitializes a pool if it has no liquidity.
+// It does so by setting the current square root price and tick to zero.
+// This is necessary for the twap to correctly detect a spot price error
+// when there is no liquidity in the pool.
+func (k Keeper) uninitializePool(ctx sdk.Context, poolId uint64) error {
```

Comment:
nit: just curious, what was the reason this was called uninitializePool instead of initializePool?